p2panda-store/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "p2panda-store"
+version = "0.1.0"
+edition = "2021"
+
+[lints]
+workspace = true
+
+[dependencies]
+serde = { version = "1.0.203" }
```

Comment:
Can we do this without serde? And if not, can we put it behind a feature flag for the in-memory variant?

Reply:
Ah, good spot, thanks :pray: this is only needed in tests, so I've moved it to `dev-dependencies` where it should have been.